modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -13607,17 +13813,17 @@ private DLFolder _getDLFolder(
 	}
 
 	private String _getEndpoint(
-		long groupId, ObjectDefinition objectDefinition) {
+		ObjectDefinition objectDefinition, Object scopeKey) {
 
 		ObjectScopeProvider objectScopeProvider =
 			_objectScopeProviderRegistry.getObjectScopeProvider(
 				objectDefinition.getScope());
 
-		if (objectScopeProvider.isGroupAware()) {
-			return objectDefinition.getRESTContextPath() + "/scopes/" + groupId;
+		if (!objectScopeProvider.isGroupAware()) {
+			return objectDefinition.getRESTContextPath();
```

Comment:
Let's avoid using negated conditions. It's generally easier to understand the logic when we write conditions in their positive form whenever possible.

Reply:
Even with our refactoring (the parameter change), we can leave the code as it is on master.